apps/labs/posts/lets-make-pygments-accessible.md

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Let's make Pygments accessible!'
+author: stephannie-jimenez
+published: November 7, 2022
```

Comment:
```suggestion
published: November 15, 2022
```

Moving a week later as we are deploying on Tuesdays and have a new post for this week